data/pages/opam-packaging.md

```diff
@@ -0,0 +1,259 @@
+    ]
+
+The last two fields are of particular interest:
+
+-   The `url` section describes where to find your package. It must have
+    at least the following fields: the `src` field with the URL of the
+    package archive, and the `checksum` field with the checksum of that
+    archive prefixed by `sha512=`.
+-   The `tags` field contains Rocq-specific metadata. See also the [RFC
+    on Opam
+    metadata](https://github.com/rocq-prover/rfcs/blob/master/text/003-opam-metadata.md).
+
+For a description of the other fields, see the [relevant section of the
+Opam
+Manual](https://opam.ocaml.org/doc/Manual.html#Package-definitions).
+
+The checksum can be obtained with:
+
+    curl -L https://github.com/$YOU/foo/releases/download/1.0.0/1.0.0.tar.gz | shasum -a 512
+
+In the `opam` file, the checksum must be prefixed with the hashing
+algorithm, here `sha512=`. (`sha256=`, and `md5=` are also supported.)
+
+Commit the new `opam` file:
+
+    git add released/packages/rocq-foo/rocq-foo.1.0.0/opam
+    git commit -m 'Package rocq-foo.1.0.0'
+
+### Test your new package
+
+The preliminary step is to lint your `opam` file as follows
+
+    opam lint --check-upstream released/packages/rocq-foo/rocq-foo.1.0.0/opam
+
+Once no more errors are given, the best way to test your package is to
+add your local clone of `rocq-prover/opam` to opam as follows, and then
+run `opam install` on your new package in verbose mode:
+
+    opam repo add test ./released
+    opam install -v rocq-foo
+
+If things go wrong, after having fixed the problem and before trying
+again to install the package, run `opam update`.
+
+### Submit your new package
+
+Submission happens by [creating a pull
+request](https://help.github.com/articles/creating-a-pull-request/) on
+the [rocq-prover/opam
+repository](https://github.com/rocq-prover/opam).
+
+First push your changes to your personal fork:
+
+    git push origin rocq-foo.1.0.0
+
+Then visit the GitHub page of your personal fork and click on the *new
+pull request* button.
+
+## Making good packages
+
+### Conventions
+
+1.  The archive follows a [layout](opam-layout.html). Regular packages
+    shall be placed in the `released` directory. One can also write
+    packages that install development branches of a software. In that
+    case `extra-dev` directory has to be used and the version has to end
+    in `dev` like `mybranch.dev`.
+2.  The package name should start with `rocq-`, for example `rocq-color`
+    or `rocq-interval`.
+3.  The `tags` field in the `opam` file can contain additional metadata
+    (like a categorization or the Rocq logical path the package
+    populates) as described in
+    [RFC3](https://github.com/rocq-prover/rfcs/blob/master/text/003-opam-metadata.md)
+
+### Rules of thumb
+
+The released repository shall contain software that works with a stable
+version of Rocq. Packages are maintained by their corresponding authors
+or by the Rocq team. Dependencies must be expressed in a conservative way
```

Comment:
Mention that opam packages should not depend on the "rocq-prover" package ([9.0 changelog](https://rocq-prover.org/doc/V9.0.0/refman/changes.html#porting-to-the-rocq-prover))?